Frontend/ui-common/src/Components/PopUpModal/Controller.purs

```diff
@@ -348,6 +358,15 @@ config = {
     , fareEstimateText : ""
     , tipSelectedText : ""
     , listViewArray : []
+    , coverVideoConfig : {
+        visibility : GONE ,
+        height : V 400 ,
+        width : WRAP_CONTENT ,
+        margin : (Margin 0 0 0 0) ,
+        padding : (Padding 0 0 0 0) ,
+        mediaType : "VideoLink",
```

Comment:
Can we remove this value? else function will be called everyTime
`pure $ setYoutubePlayer (youtubeData state.coverVideoConfig "VIDEO") id (show PLAY)`